### PR TITLE
Feature/navigation bar

### PR DESCRIPTION
**Pull request by tommytong-seekasia**
Add a simplified header for navigation.

EXAMPLE USAGE:

```js
<NavigationBar handleClick={onClick} />
```

### CODE REVIEW

File: react/NavigationBar/NavigationBar.less [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @media (min-width: 992px){
+        width: 970px;
+    }
+    @media (min-width: 1200px){
+        width: 1170px;
+    }
+}
+.toggle {
+    position: absolute;
+    top: 12px;
```
Comment (ZhenYongSeekAsia) [inline react/NavigationBar/NavigationBar.less]: better use the @base-grid, @grid-width and @grid-height to set the length. 
included other length variable.

Comment (tommytong-seekasia) [reply react/NavigationBar/NavigationBar.less]: updated to use variables

File: react/NavigationBar/NavigationBar.less [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+.root{
+    width:100%;
+    background: @saWhite;
+    clear: both;
+}
+.container { 
+    background: @saWhite;    
+    position: relative;
+    z-index:2;
+    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
```
Comment (ZhenYongSeekAsia) [inline react/NavigationBar/NavigationBar.less]: is this must use Helvetica? cause as my understanding, Muli is our main font family now. 
based on brand.seekasia.com

File: react/NavigationBar/NavigationBar.less [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    -webkit-font-smoothing: auto;   
+    margin: 0 auto; 
+    min-height: @grid-row-height * 12;
+    clear: both;
+}
+.toggle {
+    position: absolute;
+    top: @grid-row-height * 3;
+    left: @grid-gutter-width;
+    z-index:2;
```
Comment (ZhenYongSeekAsia) [inline react/NavigationBar/NavigationBar.less]: will it have effect when container and toggle also z-index:2 ?

File: react/NavigationBar/NavigationBar.less [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    padding:0;
```
Comment (ZhenYongSeekAsia) [inline react/NavigationBar/NavigationBar.less]: format issue, should be a space after colon ";"

File: react/NavigationBar/NavigationBar.less [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    width: @grid-base * 6;
+    height: @grid-base * 6;
+    border: none;
+    background: none;
+    cursor: pointer;
+    color: #1c3f94;
```
Comment (ZhenYongSeekAsia) [inline react/NavigationBar/NavigationBar.less]: can use @jsBlue from theme/palette

File: react/NavigationBar/NavigationBar.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import MenuIcon from 'seek-asia-style-guide/react/LeftArrowIcon/LeftArrowIcon';
```
Comment (ZhenYongSeekAsia) [inline react/NavigationBar/NavigationBar.js]: using LeftArrowIcon might easier to understand.

File: react/NavigationBar/NavigationBar.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+import styles from './NavigationBar.less';
+import PropTypes from 'prop-types';
+
+const NavigationBar = ({ handleClick }) => (
+  <header className={styles.root} role="banner" aria-label="Primary navigation">
+    <section className={styles.content}>
```
Comment (ZhenYongSeekAsia) [inline react/NavigationBar/NavigationBar.js]: content class not exist in navifationBar.less

File: react/NavigationBar/NavigationBar.less (hunk 4 above)
Comment (ZhenYongSeekAsia) [reply react/NavigationBar/NavigationBar.less]: can use prettier to fix this.

File: react/NavigationBar/NavigationBar.less [hunk 8]
```diff
@@ -9,7 +9,7 @@
     background: @saWhite;    
     position: relative;
     z-index:2;
```
Comment (ZhenYongSeekAsia) [inline react/NavigationBar/NavigationBar.less]: format issue.

File: react/NavigationBar/NavigationBar.less [hunk 9]
```diff
@@ -9,7 +9,7 @@
...
-    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+    font-family: Muli,"Helvetica Neue",Helvetica,Arial,sans-serif;
```
Comment (ZhenYongSeekAsia) [inline react/NavigationBar/NavigationBar.less]: if no specific purpose, actually no need to put this and default will inferit.

File: react/NavigationBar/NavigationBar.less (hunk 8 above)
Comment (ZhenYongSeekAsia) [reply react/NavigationBar/NavigationBar.less]: and seem like no effect.

**Pull request merged**